src/plugins/foxPage/components/Layout.stories.tsx

```diff
@@ -231,17 +210,7 @@ export const FoxLayout: Story = () => {
                 />
               </Stack>
               <Stack flex='1 1 0%' width='full' maxWidth={{ base: 'full', xl: 'sm' }} spacing={4}>
-                <AssetActions
-                  assetIcon={mockAsset.icon}
-                  assetSymbol={mockAsset.symbol}
-                  description={mockAsset.description}
-                  buyCTA={translate('plugins.foxPage.buyAssetOnCoinbase', {
-                    assetSymbol: mockAsset.symbol,
-                  })}
-                  sellCTA='plugins.foxPage.receive'
-                  onReceiveClick={() => null}
-                  onBuyClick={() => null}
-                />
+                {/* <AssetActions assetId={mockAsset.assetId} /> */}
```

Comment:
??

Reply:
this is ok - it's only in storybook stuff